Pivot/ViewController.swift

```diff
@@ -7,19 +7,135 @@
 //
 
 import UIKit
+import CoreMotion
+import GLKit
 
 class ViewController: UIViewController {
 
+    //MARK: Properties
+    @IBOutlet weak var rotationsTextView: UITextView!
+    @IBOutlet weak var pivotOrientationView: PivotOrientationView!
+    @IBAction func reset(_ sender: Any) {
+        total = 0
+    }
+    
+    let motionManager = CMMotionManager()       // access data from CoreMotion API
+    var timer: Timer!                           // schedules updates at regular interval from motionManager
+    var previousDeviceMotion: CMDeviceMotion?   // the device motion object from the last time the update method was called
+    var total: Float = 0                        // total rotation around axis of gravity in radians
+
+    //MARK: Overwritten Methods
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
+        motionManager.startDeviceMotionUpdates()
+        timer = Timer.scheduledTimer(timeInterval: 0.01, target: self, selector: #selector(ViewController.update), userInfo: nil, repeats: true)
     }
-
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
+    
+    func update() {
+        if let deviceMotion = motionManager.deviceMotion {
+            // if there is a previous device motion, calculate the change since the last calculation
+            if let previous = previousDeviceMotion {
+                var deltaTheta = getGravityAxisRotationDelta(deviceMotion, previous)
+                if deltaTheta.isNaN { deltaTheta = 0 }
+                total += deltaTheta
+            }
+            previousDeviceMotion = deviceMotion
+            updateUI()
+        }
     }
 
+    //MARK: Private Methods
+    
+    private func getGravityAxisRotationDelta(_ current: CMDeviceMotion,_ previous: CMDeviceMotion) -> Float {
```

Comment:
This looks good and has a great first pass decomposition, I'd challenge you to go a step further and actually abstract this all out of the view controller. I'm imagining an type which can be instantiated by a CMDeviceMotion value and exposes necessary values via interface. A call site goal could be something like:
```
total += (previousDeviceMotion?.axisRotation ?? 0) + (currentDeviceMotion.axisRotation ?? 0)
previousDeviceMotion = currentDeviceMotion
updateUI()
```